changelog.d/3786.misc

```diff
@@ -0,0 +1 @@
+Add initial implementation of new state resolution algorithm
```

Comment:
this changelog seems out of place though

Reply:
Turns out git isn't psychic and needs to be explicitly told to switch branch.

Reply:
![image](https://user-images.githubusercontent.com/3307100/44992955-30ba9400-afdc-11e8-87c3-3cea3430044d.png)